sdk/src/main/java/com/google/cloud/dataflow/sdk/util/common/Counter.java

```diff
@@ -114,6 +117,13 @@
    * @throws IllegalArgumentException if the aggregation kind is not supported
    */
   public static Counter<Integer> ints(String name, AggregationKind kind) {
+    return new IntegerCounter(Name.createWithoutStructure(name), kind);
```

Comment:
Could be `Name.withoutStructure(...)` if you wanted something shorter, but `createWithoutStructure` is fine too.